en/flight_modes_rover/manual.md

```diff
@@ -0,0 +1,88 @@
+# Manual Modes (Rover)
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/rover_manual_controls.png)
+
+The manual modes listed below provide increasing levels of autopilot support:
+
+| Mode                           | Features                                                                                                   |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------- |
+| [Manual](#manual-mode)         | Directly map stick inputs to motor commands, no closed loop control.                                       |
+| [Acro](#acro-mode)             | Closed loop yaw rate control.                                                                              |
+| [Stabilized](#stabilized-mode) | Closed loop yaw rate and yaw control.                                                                      |
+| [Position](#position-mode)     | Closed loop yaw rate, yaw, and speed control, and uses GPS coordinates for improved straight line driving. |
+
+## Manual Mode
+
+<Badge type="tip" text="Differential" /><Badge type="tip" text="Ackermann" />
+
+The _Manual_ mode stops the rover when the RC control sticks are centred.
+To manually move/drive the vehicle you move the sticks outside of the centre.
+
+- [**Differential Rover**](../frames_rover/differential_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _yaw rate_ of the vehicle.
+
+- [**Ackermann Rover**](../frames_rover/ackermann_rover.md): Moving the left-stick up/down controls the _forward speed_ and moving the right-stick left/right controls the _steering angle_ of the vehicle.
+
+::: info
+The rover does not attempt to maintain a specific orientation or compensate for external factors like slopes or uneven terrain!
+The user is responsible for making the necessary adjustments to the stick inputs to keep the rover on the desired course.
+:::
+
+## Acro Mode
+
+<Badge type="tip" text="Differential" />
+
+::: warning
+Acro mode is not supported for Ackermann rovers (only differential rovers).
+:::
+
+::: info
+This mode requires a yaw rate estimate.
```

Comment:
For MC we needed up-front information about the modes, such as requires GPS, because a vehicle might not have GPS. 
But why does a user need to know this up front? I would say it is detail you would only add if we have lots of things that are preconditions, and even so, that box would appear after the "for dummies" explanation of the mode.